frame/vesting/Cargo.toml

```diff
@@ -20,9 +20,10 @@ sp-runtime = { version = "3.0.0", default-features = false, path = "../../primit
 frame-support = { version = "3.0.0", default-features = false, path = "../support" }
 frame-system = { version = "3.0.0", default-features = false, path = "../system" }
 frame-benchmarking = { version = "3.1.0", default-features = false, path = "../benchmarking", optional = true }
+log = { version = "0.4.0", default-features = false, optional = true  }
```

Comment:
log is already in the runtime FYI, you could just import and use it normally as well. 

If it is only used in try-runtime, there's no need to do `default-features = false` then.

Reply:
If I take it out the new log I added in the main migrate (_not_ try-runtime pre,post) function errors:

```
error[E0433]: failed to resolve: use of undeclared crate or module `log`
  --> frame/vesting/src/migrations.rs:50:6
   |
50 |                     log::warn!(target: "runtime::vesting", "migration failed to move a vesting schedule into a BoundedVec");
   |                     ^^^ use of undeclared crate or module `log`
  ```

Reply:
so with adding the log to `migrate` I took the log out of the try-runtime feature and made it a non-optional dep to get that error to go away: da798992417a1074cf0da8cdf8008d21509f0149

Please advise though if it looks like I am missing something

Reply:
sounds correct to me now.